tools/condition.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2022, seL4 Project a Series of LF Projects, LLC
+#
+# SPDX-License-Identifier: BSD-2-Clause or GPL-2.0-only
+#
+
+def remove_ws_comments(node):
+    for child in list(node.childNodes):
+        if child.nodeType == node.COMMENT_NODE or (child.nodeType == node.TEXT_NODE and len(child.data.strip()) == 0):
+            node.removeChild(child)
+
+
+def condition_to_cpp(conditions):
+    n = len(conditions)
+    assert n <= 1
```

Comment:
See https://github.com/seL4/seL4/pull/789/files#r821356538, we could support more then two also.

Reply:
Ah sorry, I misinterpreted that. Maybe add a comment